base_installation.sh

```diff
@@ -111,7 +110,7 @@ if [ ! -e data/gi_taxid_prot.dmp ] ; then
   cd $metAnnotateDir
 fi
```

Comment:
Add an else statement here which echoes something like "Refeq protein gi to taxaid mapping file already cached."